app/assets/stylesheets/application.scss

```diff
@@ -1,32 +1,17 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
- * or vendor/assets/stylesheets of plugins, if any, can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the top of the
- * compiled file, but it's generally better to create a new file per style scope.
- */
-
 /* Override for Bootstrap 3 font location */
 /* Start importing Bootstrap */
-@import "bootstrap-sprockets";
```

Comment:
https://github.com/twbs/bootstrap-sass#a-ruby-on-rails

Sprockets should be loaded first ;)